.idea/misc.xml

```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```

Comment:
Revert this change too

Reply:
@kartikeysaran always make sure that you reply to comments.
In this cane it cane be `Done`

Reply:
@rt4914 okay

Reply:
I have reverted back the changes @FareesHussain thankyou for your help !